main.cc

```
@@ -1119,7 +1118,9 @@ int main(int argc, char*argv[])
 
       flag_tmp = flags["DISABLE_CONCATZ_GENERATION"];
       if (flag_tmp) disable_concatz_generation = strcmp(flag_tmp,"true")==0;
+}
 
+void preprocess(){
```

Please return an error code in this and similar functions.